docs/reports/peer_testing_1/usability_tasks.md

```diff
@@ -0,0 +1,38 @@
+# List of Usability Tasks
+
+## Users
+
+As a user, you are a member of the general public who is using the Kukarella web application; you are someone who would
+like to have a more immersive text-to-speech experience where an animated avatar reads user text aloud.
+
+As of right now, the application can extract the timings for the mouth movements, but no animation will be produced.
+
+Please attempt the following scenarios below in your pursuit of having an animated text-to-speech experience.
+
+The output file will be a json file or frames **(This may change depending on our timeline)
+
+### User Scenario 1: FileNotFound
+
+Enter no audio or text file
+
+### User Scenario 2: InvalidFileType
+
+Input a non-wav file as the audio file
+
+### User Scenario 3: CorruptedFile
+
+Input a corrupted .wav or .txt file
+
+## Developers
+
+As a developer, you are a Kukarella employee who will be responsible for incorporating the API into your current system
+and will be adding more customizable assets for the avatars in the future.
+
+Since our product is not yet wrapped into an API, nor are any frames being produced, we don't have any developer scenarios yet.
+
+Stay tuned for developer scenarios in Peer Testing II.
+
+### Developer Scenario 1: Change the Mouth Assets
+
+**(This might not be included depending on our progress this week)
+Change out the mouth assets (files) for new ones.
```

Comment:
not sure if we currently have a mouth asset directory, but some of the work on branch ffmpef-exploration includes these assets

Reply:
we will want to give the option for them to switch the mouths to something else, but I'm not sure if this is a task we want for Peer Testing I or Peer Testing II